multicluster/config/default/additional_role_patch.yaml

```diff
@@ -1,26 +1,27 @@
 # additional permissions for controller.
-- op: add
-  path: /rules/0
-  value:
-    apiGroups:
-    - admissionregistration.k8s.io
-    resources:
-    - mutatingwebhookconfigurations
-    - validatingwebhookconfigurations
-    resourceNames:
-    - antrea-multicluster-mutating-webhook-configuration
-    - antrea-multicluster-validating-webhook-configuration
-    verbs:
-    - get
-    - update
-- op: add
-  path: /rules/0
-  value:
-    apiGroups:
-    - ""
-    resources:
-    - secrets
-    verbs:
-    - get
-    - list
-    - watch
\ No newline at end of file
+# TODO: webhooks are disabled for now, reenable it after
+#- op: add
+#  path: /rules/0
+#  value:
+#    apiGroups:
+#    - admissionregistration.k8s.io
+#    resources:
+#    - mutatingwebhookconfigurations
+#    - validatingwebhookconfigurations
+#    resourceNames:
+#    - antrea-multicluster-mutating-webhook-configuration
+#    - antrea-multicluster-validating-webhook-configuration
+#    verbs:
+#    - get
+#    - update
+#- op: add
```

Comment:
I think we need keep following secret patch since member cluster need it to read secret to access leader cluster.